mix.exs

```diff
@@ -55,12 +55,13 @@ defmodule Membrane.AAC.FDK.Plugin.MixProject do
 
   defp deps do
     [
-      {:bunch, "~> 1.3"},
-      {:membrane_core, "~> 0.10.0"},
-      {:membrane_common_c, "~> 0.13.0"},
+      {:bunch, "~> 1.4"},
```

Comment:
There's no need to require that, it may possibly break compatibility if something specifies "~> 1.4.0". 1.4 is required by relying on `Bunch.Typespec.enum_to_alternative`